extensions/interactions/FractionInput/FractionInputValidationService.js

```diff
@@ -34,15 +34,15 @@ oppia.factory('FractionInputValidationService', [
         var toFloat = function(fraction) {
           return FractionObjectFactory.fromDict(fraction).toFloat();
         };
-        /*
-        Store an answer range for every rule, then check for redundant
-        ranges. A range is an object of the form:
-        {
-          lb: float, lower bound
-          ub: float, upper bound
-          lbi: bool, is lower bound inclusive
-          ubi: bool, is upper bound inclusive
-        }
+        /**
+        * Store an answer range for every rule, then check for redundant
```

Comment:
Nit: indent this line, up to line 45, by 1, so that the * matches the middle star in line 37.